Serverless docs polish (#880)
neo4j/graph-data-science-client: doc/modules/ROOT/pages/graph-analytics-serverless.adoc

:page-aliases: gds-session

Aura Graph Analytics Serverless is an on-demand ephemeral compute environment for running GDS workloads.
Each compute unit is called a _GDS Session_.
It is offered as part of link:https://neo4j.com/docs/aura/graph-analytics/[Neo4j Aura], a fast, scalable, always-on, fully automated cloud graph platform.

NOTE: Aura Graph Analytics Serverless is not available by default.
See the Aura https://neo4j.com/docs/aura/graph-analytics/#aura-gds-serverless[docs] for details on how to enable it.

> A GDS Session is in one of three flavours, determined by its data source:

Reviewer: ```suggestion
There are three types of GDS Sessions:
```

> * *Attached*, sourced from a Neo4j AuraDB instance.
> * *Self-managed*, sourced from a self-managed Neo4j DBMS.
> * *Standalone*, sourced from a non-Neo4j data source.

Reviewer: ```suggestion
* *Attached*: the data source is a Neo4j AuraDB instance.
* *Self-managed*: the data source is a self-managed Neo4j DBMS.
* *Standalone*: the data source is not based on Neo4j.
```

> We call the process of populating the session with data a _remote projection_.

Reviewer: ```suggestion
The process of populating the session with data is called _remote projection_.
```

CLIENT_ID = "my-aura-api-client-id"
CLIENT_SECRET = "my-aura-api-client-secret"
PROJECT_ID = None

# Create a new GdsSessions object
sessions = GdsSessions(api_credentials=AuraAPICredentials(CLIENT_ID, CLIENT_SECRET, PROJECT_ID))
----

> All available methods and parameters are listed in our API https://neo4j.com/docs/graph-data-science-client/{docs-version}/api/sessions/gds_sessions/[reference].

Reviewer: ```suggestion
All available methods and parameters are listed in the https://neo4j.com/docs/graph-data-science-client/{docs-version}/api/sessions/gds_sessions/[API reference].
```

As a convention, always use the variable name `gds` for the return value of `get_or_create()`.

==== Session expiration and deletion

> When the session is created, an optional `ttl` parameter can be configured, which specifies how quickly an inactive session will expire.

Reviewer: ```suggestion
When the session is created, an optional `ttl` parameter can be configured to set the time after which an inactive session will expire.
```

The default value for `ttl` is 1 hour and the maximum allowed value is 7 days.
An expired session cannot be used to run workloads, does not cost anything, and will be deleted automatically after 7 days.
It can also be deleted through the Aura Console UI.

> ===== Maximum lifetime

Reviewer: ```suggestion
==== Maximum lifetime
```

I think we don't need a deeper section level for this.
